test/functional/core/startup_spec.lua

```diff
@@ -524,8 +524,8 @@ describe('sysinit', function()
     screen:expect([[
       ^                                                            |
       Entering Debug mode.  Type "cont" to continue.              |
-      pre-vimrc command line                                      |
-      cmd: set noruler                                            |
+      {MATCH:.+}|
+      {MATCH:cmd: .+}|
```

Comment:
can keep this test somewhat explicit, gives us some visibility.

Reply:
`-D` prints the first VimScript command along with its source. After this PR it becomes `aunmenu *` (as default mappings precede even `--cmd`). But even this may change some day. So I'm not sure what could be matched here without future issues? Or maybe we should move `--cmd` to occur _before_ default mappings?

Reply:
I'd say just add `aunmenu *` here. It's not too bad to update it every now and then, and it gives visbility.

Reply:
Done